### PR TITLE
Fixing typos in documentation

### PR DESCRIPTION
**Pull request by arne-da**
This draft pull request is collecting typos as I go through the tutorials. I will convert this from a draft pull request once I have collected multiple typos/have gone through most of the tutorials.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags

### CODE REVIEW

**Comment by arne-da** [comment]
This is a silly question but do I need to squash my commits locally? Usually I can do it here on Github but I'm not seeing it right now.

**Comment by garyverhaegen-da** [comment]
> This is a silly question but do I need to squash my commits locally? Usually I can do it here on Github but I'm not seeing it right now.

You don't _need_ to. The repo is set up to always "squash & merge", so every PR becomes a single commit on master. However, ensuring your PR only has one commit to begin with gives you slightly better control over the final commit (e.g. commit message, date, etc.)

GitHub doesn't let you squash at random points, it's only an option at the time of merging. You will not be able to merge until the PR is approved and has passed all CI tests. In this case, because you opened the PR from a fork, CI tests are not run automatically.

I've sent you an invitation to the digital-asset team, so for future PRs you should be able to push your branches directly to this repo. This should make things a bit easier. You'll also now be able to trigger CI runs with the magic words `/azp run`, so if you end up modifying this PR you can at least get CI to run on it yourself.

File: docs/source/app-dev/app-arch.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ architecture, providing you with an excellent starting point for your own applic
 
 Backend
 ~~~~~~~
-
+
```
Comment (bame-da) [inline docs/source/app-dev/app-arch.rst:35]:
```suggestion

```

File: docs/source/app-dev/app-arch.rst [hunk 2]
```diff
@@ -136,9 +136,9 @@ The interaction of a Daml application with the ledger is inherently asynchronous
 There are several things that can fail during this time window: the application can crash, the participant node can crash, messages can be lost on the network, or the ledger may be just slow to respond due to a high load.
 
 If you want to make sure that a command is not executed twice, your application needs to robustly handle all the various failure scenarios.
-Daml ledgers provide a mechanism for :ref:`command deduplication <command-submission-service-deduplication>` to help deal this problem.
+DAML ledgers provide a mechanism for :ref:`command deduplication <command-submission-service-deduplication>` to help deal with this problem.
```
Comment (bame-da) [inline docs/source/app-dev/app-arch.rst:139]:
```suggestion
Daml ledgers provide a mechanism for :ref:`command deduplication <command-submission-service-deduplication>` to help deal with this problem.
```

File: docs/source/daml/intro/2_DamlScript.rst [hunk 3]
```diff
@@ -58,10 +58,10 @@ Running scripts
 
 There are a few ways to run Daml Scripts:
 
-- In Daml Studio against a test ledger, providing visualizations of the resulting ledger
-- Using the command line ``daml test`` also against a test ledger, useful for continuous integration
-- Against a real ledger, take a look at the documentation for :doc:`Daml Script </daml-script/index>` for more information.
-- Interactively using :doc:`Daml REPL </daml-repl/index>`.
+- In DAML Studio against a test ledger, providing visualizations of the resulting ledger.
```
Comment (bame-da) [inline docs/source/daml/intro/2_DamlScript.rst:61]: Daml is now title case. please revert.

**Review by bame-da — APPROVED** [review]
Approved with a few minor change requests. Please address those before merging.

**Comment by bame-da** [comment]
And sorry for leaving this so long. I've only just noticed how long this has been open 😱 .

**Comment by cocreature** [comment]
> And sorry for leaving this so long. I've only just noticed how long this has been open scream .

It was a draft until 13h ago so don’t feel too bad :slightly_smiling_face:

File: docs/source/daml/intro/2_DamlScript.rst (hunk 3 above)
Comment (arne-da) [reply docs/source/daml/intro/2_DamlScript.rst]: Added

**Comment by arne-da** [comment]
Thank you for the helpful answers/write rights/the review. I've fixed the small issues

**Pull request merged**